src/components/AddItemForm.spec.js

```diff
@@ -1,14 +1,15 @@
-/*global test*/
-
 import React from 'react'
 import AddItemForm from './AddItemForm.js'
+import AppBar from '@material-ui/core/AppBar'
+import TextField from '@material-ui/core/TextField'
+import Button from '@material-ui/core/Button'
 import jesttestsetup from '../../test/jestsetup.js'
 
 describe('AddItemForm', () => {
 
-  const wrapper = shallow(<AddItemForm/>)
-  const textBox = wrapper.find('.addToDoInput')
-  const button = wrapper.find('.addToDoButton')
+  const wrapper = mount(<AddItemForm/>)
+  const textBox = wrapper.find(TextField)
+  const button = wrapper.find(Button)
```

Comment:
it's also convention to do the `mount/render/shallow` stuff inside of the `it` blocks, just to maintain a bit more of a clean test in every assertion.